src/cljs/athens/events.cljs

```diff
@@ -945,7 +945,7 @@
 
 (reg-event-fx
   :block/save
-  (fn [_ [_ {:keys [uid old-string new-string callback] :as args}]]
+  (fn [_ [_ {:keys [uid old-string new-string callback add-time?] :as args}]]
```

Comment:
I did not add the `add-time?` key to all the other places from where this event is called because this value will be `nil` if not passed and that would also work fine for this event. Is this right?

Reply:
What if I did this : 

```
{:keys [block new-uid embed-id add-time?]
 :or   {add-time? false}
 :as args}
```
then I think we don't need to add multi-arity in the common event and remote event because `add-time?` value is always passed as boolean and not `nil`

Reply:
This will also work as goo